containers/install-ansible-container.yml

```diff
@@ -0,0 +1,20 @@
+---
+- hosts: localhost
+  tasks:
+    - name: 'Create .tmp directory'
+      file:
+        path: .tmp
+        state: directory
+
+    - name: 'Clone ansible-container'
+      git:
+        repo: https://github.com/ehelms/ansible-container.git
+        version: support-secrets-vault
+        dest: .tmp/ansible-container
+        update: yes
+
+    - name: 'Install ansible-container'
+      become: true
```

Comment:
You could use ```pip3 install --user --editable .[docker,openshift]``` and drop the become: true. You do need to be aware that it places the binary in ```~/.local/bin``` which may not be in your ```$PATH```. Maybe you could also use ```.tmp/ansible-container``` instead of ```.``` and drop the ```chdir```.